boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -115,6 +115,17 @@
 	pinctrl-names = "default";
 	status = "okay";
 	clock-frequency = <I2C_BITRATE_FAST>;
+
+	stmpe811@41 {
```

Comment:
neat, add a stmpe811 nodelabel here and a

```
       lvgl_pointer {
               compatible = "zephyr,lvgl-pointer-input";
               input = <&stmpe811>;
               invert-x;
               invert-y;
       };
```

in the block above (after `aliases`) and this will work with lvgl out of the box

Reply:
Thanks, this is now addded.